wgpu-core/src/instance.rs

```diff
@@ -176,7 +176,19 @@ pub struct Adapter<A: hal::Api> {
 }
 
 impl<A: HalApi> Adapter<A> {
-    fn new(raw: hal::ExposedAdapter<A>) -> Self {
+    fn new(mut raw: hal::ExposedAdapter<A>) -> Self {
+        // WebGPU requires this offset alignment as lower bound on all adapters.
+        const MIN_BUFFER_OFFSET_ALIGNMENT_LOWER_BOUND: u32 = 32;
+
+        let limits = &mut raw.capabilities.limits;
+
+        limits.min_uniform_buffer_offset_alignment = limits
+            .min_uniform_buffer_offset_alignment
+            .min(MIN_BUFFER_OFFSET_ALIGNMENT_LOWER_BOUND);
```

Comment:
Did you mean `max`? will make it so the alignment is at most 32.

Reply:
Gah, I also got fooled by that. Petition for min/max to be "no_more_than" and "no_less_than"

Reply:
egui does that actually because of that :D

Reply:
excellent catch btw. Fixed now